omod/src/main/java/org/openmrs/module/appframework/rest/ExtensionResource.java

```diff
@@ -104,4 +112,28 @@ protected PageableResult doSearch(RequestContext context) {
         }
         return new NeedsPaging<Extension>(results, context);
     }
+
+    @Override
+    public Model getGETModel(Representation rep) {
+        ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+
+        if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
+            modelImpl
+                    .property("uuid", new StringProperty())
+                    .property("appId", new StringProperty())
+                    .property("extensionPointId", new StringProperty())
+                    .property("type", new StringProperty())
+                    .property("label", new StringProperty())
+                    .property("url", new StringProperty())
+                    .property("icon", new StringProperty())
+                    .property("order", new IntegerProperty())
+                    .property("requiredPrivilege", new StringProperty())
+                    .property("featureToggle", new StringProperty())
+                    .property("require", new StringProperty())
+                    .property("script", new StringProperty())
+                    .property("extensionParams", new MapProperty(new StringProperty()))
+                    .property("belongsTo", new RefProperty("ExtensionGet"));
```

Comment:
`belongsTo` is not an Extension resource. So you may just use `new RefProperty()`